File: ts-sdk/src/sdk/models/operations/completeagateddeployrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CompleteAGatedDeployRequestPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}


export class CompleteAGatedDeployRequestRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CompleteAGatedDeployRequestPathParams;
}


export class CompleteAGatedDeployRequestResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  completeAGatedDeployRequest200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/completeagateddeployrequest.go
package operations

type CompleteAGatedDeployRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type CompleteAGatedDeployRequestRequest struct {
	PathParams CompleteAGatedDeployRequestPathParams
}

type CompleteAGatedDeployRequestResponse struct {
	ContentType                                         string
	StatusCode                                          int64
	CompleteAGatedDeployRequest200ApplicationJSONObject map[string]map[string]interface{}
}
